### PR TITLE
bug(Alert): changed success icon color and success background color.

### PR DESCRIPTION
**Pull request by jameslau**
In this change I made the global green color variable.

### CODE REVIEW

**Comment by jameslau** [comment]
hey @michael-coker look at the committed code.

File: src/patternfly/components/Alert/styles.scss [hunk 1]
```diff
@@ -24,8 +24,8 @@
   --pf-c-alert__action--PaddingLeft: var(--pf-global--spacer--lg);
 
   // Success
-  --pf-c-alert--m-success__icon--Color: var(--pf-global--success-color--200);
-  --pf-c-alert--m-success__icon--BackgroundColor: var(--pf-global--success-color--100);
+  --pf-c-alert--m-success__icon--Color: var(--pf-global--success-color--100);
+  --pf-c-alert--m-success__icon--BackgroundColor: var(--pf-global--success-color--200);
```
Comment (michael-coker) [inline src/patternfly/components/Alert/styles.scss]: Here you need to use color--500.

File: src/patternfly/components/Alert/styles.scss [hunk 2]
```diff
@@ -24,8 +24,8 @@
   --pf-c-alert__action--PaddingLeft: var(--pf-global--spacer--lg);
 
   // Success
-  --pf-c-alert--m-success__icon--Color: var(--pf-global--success-color--200);
```
Comment (michael-coker) [inline src/patternfly/components/Alert/styles.scss]: this thing too

File: src/patternfly/components/Alert/styles.scss (hunk 1 above)
Comment (michael-coker) [reply src/patternfly/components/Alert/styles.scss]: Ok I made that change.

Comment (jameslau) [reply src/patternfly/components/Alert/styles.scss]: I will submit the new changes.

File: src/patternfly/components/Alert/styles.scss (hunk 2 above)
Comment (michael-coker) [reply src/patternfly/components/Alert/styles.scss]: fixed it.

**Comment by michael-coker** [comment]
Just testing.

**Pull request closed without merging**